.goreleaser.yml

```
@@ -525,6 +525,8 @@ docker_manifests:
 checksum:
   name_template: "{{ .ProjectName }}-v{{ .Version }}-SHA256SUMS"
   algorithm: sha256
+  extra_files:
+    - glob: "./observiq-otel-collector.msi*"
```

I expected this to fail locally, but it did not. The file just omitted the msi because I do not have it.

I think we can keep the signature file out of the checksum file. The file does not contain other sig files.
```suggestion
  extra_files:
    - glob: "./observiq-otel-collector.msi"
```

---

Oh yeah I don't know why I added it good call.